.vscode/tasks.json

```diff
@@ -0,0 +1,20 @@
+// See https://go.microsoft.com/fwlink/?LinkId=733558
+// for the documentation about the tasks.json format
```

Comment:
<!--
  7 failure:  " should be ' lin..., " should be ' lin..., " should be ' lin..., Unnecessarily quo..., Unnecessarily quo..., tab indentation e..., unused expression...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: .vscode/tasks.json;
  Line: 2;
-->
- :no_entry_sign: " should be ' line - package.json (quotemark)
- :no_entry_sign: " should be ' line - package.json (quotemark)
- :no_entry_sign: " should be ' line - tslint-with-errors.json (quotemark)
- :no_entry_sign: Unnecessarily quoted property 'rules' found. line - tslint-with-errors.json (object-literal-key-quotes)
- :no_entry_sign: Unnecessarily quoted property 'version' found. line - package.json (object-literal-key-quotes)
- :no_entry_sign: tab indentation expected line - package.json (indent)
- :no_entry_sign: unused expression, expected an assignment or function call line - tasks.json (no-unused-expression)